components/generic/OptimisedImage.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <span>
```

Comment:
Is the span necessary? It breaks the image compare. If possible remove it here, or else, the image compare code should be slightly updated.  

```diff --git a/components/generic/CompareImageSlider.vue b/components/generic/CompareImageSlider.vue
index 77221612..a0eda8d4 100644
--- a/components/generic/CompareImageSlider.vue
+++ b/components/generic/CompareImageSlider.vue
@@ -246,7 +246,7 @@
     position: relative;
     overflow: hidden;
 
-    img {
+    span {
       width: 100%;
 
       &:nth-child(1) {
```

Reply:
It seems not. I will remove it.